src/main/java/steps/LoginSteps.java

```diff
@@ -0,0 +1,51 @@
+package steps;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import lombok.extern.log4j.Log4j2;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import pages.LoginPage;
+import utils.PropertyReader;
+
+@Log4j2
+public class LoginSteps {
+    public static final String BASE_URL = "https://lada29.testrail.io/index.php?/auth/login";
+    private LoginPage loginPage;
+    WebDriver driver;
+
+    @Before
+    public void init() {
+        WebDriverManager.chromedriver().setup();
+        driver = new ChromeDriver();
+    }
+
+    @Given("Open site TestRail")
+    public void openSiteTestRail() {
+        loginPage = new LoginPage(driver);
+        log.info("Open url: " + BASE_URL);
+        driver.get(BASE_URL);
+    }
+
+    @When("Filling fields with email and password")
```

Comment:
@When("User filling in standard email and password on Login page")

Reply:
fixed